src/core-layers/line-layer/line-layer.js

```diff
@@ -167,10 +167,10 @@ export default class LineLayer extends Layer {
     for (const object of data) {
       const sourcePosition = getSourcePosition(object);
       const targetPosition = getTargetPosition(object);
-      value[i + 0] = fp64ify(sourcePosition[0])[1];
-      value[i + 1] = fp64ify(sourcePosition[1])[1];
-      value[i + 2] = fp64ify(targetPosition[0])[1];
-      value[i + 3] = fp64ify(targetPosition[1])[1];
+      value[i + 0] = fp64LowPart(sourcePosition[0]);
```

Comment:
Wonder if could save a fair amount of library size by not duplicating this code in every layer?